conf/CODE.public.conf

```diff
@@ -17,3 +17,21 @@ googleAuth.redirectUrl = "https://support.code.dev-theguardian.com/oauth2callbac
 paymentApi.url="https://payment.code.dev-guardianapis.com"
 membersDataService.api.url="https://members-data-api.code.dev-theguardian.com"
 play.filters.headers.contentSecurityPolicy = "default-src 'self' polyfill.guim.co.uk www.paypalobjects.com www.paypal.com www.sandbox.paypal.com js.stripe.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io data: wss: 'unsafe-inline' q.stripe.com payment.code.dev-guardianapis.com"
+
+// ------------------------ Feature switches ----------------------------
+// To disable a feature change the value from On to Off and redeploy the app.
+// To override a value for local development change it in your private conf file
+// /etc/gu/support-frontend.private.conf and that value will be used instead.
+switches {
+  //Payment methods for one-off contributions
+  oneOff {
+    stripe=On
+    payPal=On
```

Comment:
We should add a comment  here to indicate 24/7 how to disbale PayPal one-off flow.

Reply:
Maybe only in the PROD conf.